src/IsoCodes/Bsn.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace IsoCodes;
+
+/**
+ * Class Bsn.
+ *
+ * In the Netherlands, the citizen service number (BSN) is a unique personal number
+ * allocated to everyone registered in the Municipal Personal Records Database
+ */
+class Bsn implements IsoCodeInterface
```

Comment:
I think this class should not be extended and should be `final`. cc @ronanguilloux